pandas/io/parsers/readers.py

```diff
@@ -489,18 +459,6 @@ class _DeprecationConfig(NamedTuple):
     msg: str | None
 
 
-_deprecated_defaults: dict[str, _DeprecationConfig] = {
```

Comment:
It might be worth leaving this in, or leaving a comment linking back to this line in this PR.
This is a nice way to do the deprecations, and I think there's a quite a few keywords left for read_csv that can be combined and deprecated.

Reply:
Added a comment